.github/workflows/focus-ios-smoke

```diff
@@ -0,0 +1,124 @@
+name: "Focus/Firefox Tests"
+    
+on:
+    workflow_dispatch:
+
+env:
```

Comment:
I have taken the possible parameters of the future reusable workflows as `env`: `browser`, `ios_version`, `ios_simulator`, `xcode_test_plan`, `xcode_scheme` and `xcode_target`.